web/js/map-OpenLayers.js

```diff
@@ -90,6 +91,34 @@ function fms_markers_list(pins, transform) {
     return markers;
 }
 
+function fms_marker_size_for_zoom(zoom) {
+    // zoomOffset seems to change between pages, so check that the zoom level
+    // we're working with is in the expected range
```

Comment:
`fixmystreet.map.getZoom()` seems to include `zoomOffset` on `/around` but not on `/reports`. I couldn't figure out what was going on and this fixed the problem. Will have another look into it and see if I can get rid of this.